### ISSUE TITLE
Grammar issue.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#603]
```
Please describe the issue:
Grammar issue.

Please provide any additional information below.
"Browsers error tolerance"

Should be:

"Browsers' error tolerance"


```

Original issue reported on code.google.com by `alexzava...@gmail.com` on 18 Aug 2011 at 5:11

**Comment by GoogleCodeExporter** [comment]
```
Where is this?
```

Original comment by `ericbidelman@chromium.org` on 9 Sep 2011 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
http://www.html5rocks.com/en/tutorials/internals/howbrowserswork/
```

Original comment by `alexzava...@gmail.com` on 9 Sep 2011 at 12:41

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision f9a6b5fdf0d3.
```

Original comment by `ericbide...@html5rocks.com` on 14 Feb 2012 at 10:25
- Changed state: **Fixed**
